app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -123,6 +123,22 @@ def index_field_link(options)
       safe_join(links, ", ")
     end
 
+    # A Blacklight helper_method
+    # @param [Hash] options from blacklight invocation of helper_method
+    # @return the html_safe facet links separated by " > " to indicate facet hierarchy
+
+    def link_to_each_facet_field(options)
+      raise ArgumentError unless options[:config] && options[:value] && options[:config][:helper_facet]
+
+      separator = options[:config][:separator] ? options[:config][:separator] : ">"
+      facet_search = options[:config][:helper_facet]
```

Comment:
This seems to be required, so let's do `options.fetch(:config).fetch(:helper_facet)`